app/assets/src/components/ui/labels/PathogenLabel.jsx

```diff
@@ -0,0 +1,24 @@
+import Label from "./Label";
+import React from "react";
+
+const PathogenLabel = ({ type, number }) => {
+  let type2text = {
+    category_A: "priority A pathogen",
+    category_B: "priority B pathogen",
+    category_C: "priority C pathogen"
+  };
+  let text = type2text[type];
+  if (number) {
```

Comment:
Just compare number >1 here.
the extra check in the line below is not necessary.

Reply:
resolved because I removed the `number` prop